docs/admin/scale.md

```diff
@@ -106,12 +106,13 @@ For example, to support 120 concurrent workspace builds:
 > Note: the below information is for reference purposes only, and are not
 > intended to be used as guidelines for infrastructure sizing.
 
-| Environment      | Coder CPU | Coder RAM | Coder Replicas | Database         | Users | Concurrent builds | Concurrent connections (Terminal/SSH) | Coder Version | Last tested  |
-| ---------------- | --------- | --------- | -------------- | ---------------- | ----- | ----------------- | ------------------------------------- | ------------- | ------------ |
-| Kubernetes (GKE) | 3 cores   | 12 GB     | 1              | db-f1-micro      | 200   | 3                 | 200 simulated                         | `v0.24.1`     | Jun 26, 2023 |
-| Kubernetes (GKE) | 4 cores   | 8 GB      | 1              | db-custom-1-3840 | 1500  | 20                | 1,500 simulated                       | `v0.24.1`     | Jun 27, 2023 |
-| Kubernetes (GKE) | 2 cores   | 4 GB      | 1              | db-custom-1-3840 | 500   | 20                | 500 simulated                         | `v0.27.2`     | Jul 27, 2023 |
-| Kubernetes (GKE) | 2 cores   | 4 GB      | 2              | db-custom-2-7680 | 1000  | 20                | 1000 simulated                        | `v2.2.1`      | Oct 9, 2023  |
+| Environment      | Coder CPU | Coder RAM | Coder Replicas | Database          | Users | Concurrent builds | Concurrent connections (Terminal/SSH) | Coder Version | Last tested  |
+| ---------------- | --------- | --------- | -------------- | ----------------- | ----- | ----------------- | ------------------------------------- | ------------- | ------------ |
+| Kubernetes (GKE) | 3 cores   | 12 GB     | 1              | db-f1-micro       | 200   | 3                 | 200 simulated                         | `v0.24.1`     | Jun 26, 2023 |
+| Kubernetes (GKE) | 4 cores   | 8 GB      | 1              | db-custom-1-3840  | 1500  | 20                | 1,500 simulated                       | `v0.24.1`     | Jun 27, 2023 |
+| Kubernetes (GKE) | 2 cores   | 4 GB      | 1              | db-custom-1-3840  | 500   | 20                | 500 simulated                         | `v0.27.2`     | Jul 27, 2023 |
+| Kubernetes (GKE) | 2 cores   | 8 GB      | 2              | db-custom-2-7680  | 1000  | 20                | 1000 simulated                        | `v2.2.1`      | Oct 9, 2023  |
+| Kubernetes (GKE) | 4 cores   | 16 GB     | 2              | db-custom-8-30720 | 2000  | na. (provisioned) | 2000 simulated                        | `v2.8.4`      | Feb 28, 2024 |
```

Comment:
We've successfully used 50 concurrent builds in our 2k user scaletests, I think it'd be fair to just drop that number here.